GETTING_STARTED.md

```diff
@@ -0,0 +1,169 @@
+registries, and allows for further experimentation with arbitrary file types
+within the context of this stable andd well-defined capability.
+
+**Implementors** own projects outside the OCI for which they have determined
+value in being "OCI compliant"; whether that's a registry, a unique container
+runtime, or an end-user tool that talks to registries. They take the OCI
+specifications and implement support for them in their project, and potentially
+will use the conformance suite within the OCI to validate their compliance to
+the specification(s).
+
+**End Users** tend to gain value from the OCI specifications in an indirect
+way: they will expect projects and products that claim OCI compliance to
+interoperate smoothly with other projects and products which are OCI compliant.
+They will look to the OCI to continue maturing conformance and specifications
+to best support the cloud native ecosystem's goal of interoperability for
+runtimes, images, and distributing images and artifacts across clouds and
+platforms.
+
+## What Types of Projects Exist within the OCI?
+
+There has been some growth in the nature and use of the OCI since those
+initial meetings around the image and runtime specification in 2015. The
+following subsections define project categories which exist in the OCI today.
+
+### Specifications
+
+Clearly the image, runtime, and distribution specifications are the key
+reason for the existence of the OCI today. These standards are meant to
+provide a baseline to follow for implementors of runtimes, registries, and
+tools which interact with container images and registries.
+ - [Runtime spec](https://github.com/opencontainers/runtime-spec)
+ - [Image spec](https://github.com/opencontainers/image-spec)
+ - [Distribution spec](https://github.com/opencontainers/distribution-spec)
+
+Artifacts is not a standalone specification, but fits within this category
+as it is a repository which serves as the extension point for media types
+which include the core media types found within the image spec, as well as
+connecting those and additional media types with the distribution spec as
+the means for storage, query, and delivery of a growing list of well-defined
+artifact types defined within the project.
+ - [Artifacts](https://github.com/opencontainers/artifacts)
+
+### Spec Conformance Test
+
+Conformance tests should provide a clear, end-user runnable test suite for
+implementors to use to determine and demonstrate that an implementing project
+or product meets the explicit definitions of the specification.
+ - [OCI Conformance](https://github.com/opencontainers/oci-conformance)
+
+The most advanced conformance implementation to date is for the new distribution specification. Additional work on image and runtime conformance is ongoing.
+
+### Libraries 
+
+While hosting library code is not a common goal of the OCI, there are a few
+specific cases where small, non-end user focused, and tightly scoped libraries
```

Comment:
Defining "small" here is kind of difficult. I would like to see language here specifying that these libraries should have no or minimal dependencies. If a library is depending on an even larger project or library, it can't really be "small". I think the two libraries below are good examples of "no or minimal dependencies".

Reply:
I somewhat agree with putting more text about what "small" means, but I don't think this needs to be a legal document in terms of specificity. After all, the TOB decides whether a project meets the criteria we outline here and in the charter text.

I would also point out that in my view, what defines a library as small is its scope (though that is covered by "tightly scoped" so maybe this "small" does refer to size). As an obvious example, glibc doesn't have any dependencies, but it isn't a small library.